run_pacta_data_preparation.R

```diff
@@ -243,7 +243,8 @@ factset_entity_id__ar_company_id <-
   select(
     factset_entity_id = "factset_id",
     ar_company_id = "company_id"
-  )
+  ) %>%
+  distinct()
```

Comment:
NIT: Should this be something we warn about? like "X input contains duplicate rows, removing"
Maybe not important.

Reply:
🤷 maybe? I think that it should be expected that all of these inputs do not have random duplicate lines. 

I've already gone down the rabbit hole of imaging adding validation functions to pacta.data.preparation for all of these inputs and all of the configuration options 😈

Reply:
yeah i mean, in my perfect world, once https://github.com/RMI-PACTA/workflow.data.preparation/issues/94 is closed, I would imagine just adding a simple "warning" message in the wrapper function whenever this kind of wonkiness happens would be sufficient